### DISCUSSION TITLE
France Group order from JLCPCB

### DISCUSSION THREAD

**Discussion by nyok92** [rbaron/b-parasite#112] [General]
Is there some people interested to share costs in France/EU ?  the more we are the cheaper it is :)

**Comment by progl0r** [comment]
I would like to order 5 to 10 pieces (Germany). Never ordered a PCB before. Could you give some information about the costs?

**Comment by Alcoinus** [comment]
If you haven't ordered yet, I also might be interested in a few additional parasites to Germany.

**Comment by harmpter** [comment]
Yeah I'd also be interested in a sort of group order to the EU/Germany. I've never ordered PCBs and have no experience soldering anything that tiny. We should probably link up, like on discord or something :)

**Comment by kenny00111** [comment]
If it's not too late I would like to join as well

**Comment by BenRoe** [comment]
I still have a few left over from my first order.
Can flash the Zigbee or Bluetooth firmware if needed.
If it’s fine here is the [link](https://www.kleinanzeigen.de/s-anzeige/b-parasite-pflanzensensor-zigbee-bluetooth-home-assistant/2634913680-89-4257).
If not, let me know and i will delete the link.

**Comment by nyok92** [comment]
Hi @BenRoe , 
I am interested to buy some  units. It seems kleinanzeigen.de is not in french. 
Do you have another link to order the units in Francce ?
Thanks

**Reply by BenRoe** [reply]
Hi,
someone was already faster and is from my hometown.
I will let you know, if i do a second order.
@rbaron are you fine with it?
